test/tstDistributedTreeSpatial.cpp

```diff
@@ -121,40 +121,40 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(empty_tree_spatial, DeviceType,
     BOOST_TEST(ArborX::Details::equals(tree.bounds(), {}));
 
     ARBORX_TEST_QUERY_TREE(ExecutionSpace{}, tree,
-                           makeIntersectsBoxQueries<DeviceType>({}),
+                           makeIntersectsQueries<DeviceType>({}),
```

Comment:
What query type does it deduce here?  Should we be more explicit?

Reply:
Good catch. Actually, we have to. For some reason, that test was commented out since 67c945c8b, which seems like an oversight.